code/game/turfs/floor_ground.dm

```diff
@@ -79,10 +79,6 @@
 	mediumxenofootstep = FOOTSTEP_SAND
 	minimap_color = MINIMAP_DIRT
 
-/turf/open/floor/plating/ground/mars/random/Initialize(mapload)
-	. = ..()
-	dir = pick(GLOB.alldirs)
```

Comment:
Is this not going to impact the turf? (unless its duplicate code)

Reply:
thought its a bad copypaste because its unused, but fixed now